homeassistant/components/google/calendar.py

```diff
@@ -554,7 +558,9 @@ def _get_calendar_event(event: Event) -> CalendarEvent:
     return CalendarEvent(
         uid=event.ical_uuid,
         recurrence_id=event.id if event.recurring_event_id else None,
-        rrule=event.recurrence[0] if len(event.recurrence) == 1 else None,
+        rrule=event.recurrence[0].lstrip(RRULE_PREFIX)
+        if len(event.recurrence) == 1
+        else None,
```

Comment:
Multiline ternary operator expressions are hard to read. We prefer regular if/else blocks in that case.

Reply:
Fixed.